omnigibson/utils/usd_utils.py

```diff
@@ -746,6 +747,9 @@ def mark_valid(cls):
     @classmethod
     def _refresh(cls):
         if og.sim is not None and not cls.VALID:
+            # Check that no reads from PoseAPI are happening during a physics step.
+            assert not og.sim.currently_stepping, "Cannot refresh poses during a physics step!"
```

Comment:
what's the reason for this? I assume this would possibly only occur during the physics callback, i.e.: the robot controller callabck code?